tests/drivers/adc/adc_api/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/dt-bindings/adc/stm32_adc.h>
```

Comment:
Let's move this to `dts/arm/st/x*/stm32x*.dtsi` files

Reply:
moving the zephyr/dt-bindings/adc/stm32_adc.h from the test overlay file to the stm32x.dtsi for stm32u5x and stm32l5 series.

Reply:
Why not extending to all other STM32 series ?

Reply:
added to all the stm32 soc